Liz

lizjegede opened the pull request:
Added two stories, also included resources, google forms and an "about us" section. I will probably add some more content to the resources section.

lizjegede committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Liz

lizjegede committed:
Merge branch 'Liz' of https://github.com/madeleinel/OurStory into Liz

lizjegede committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Liz

madeleinel committed:
Merge branch 'master' into Liz

madeleinel commented:
I've resolved the issues and:
- Edited the Maya Angelou quote section so that the source is displayed correctly
- Merged the location categories "South East England"  with "Scotland" for consistency

madeleinel commented:
I've resolved the issues and:
- Edited the Maya Angelou quote section so that the source is displayed correctly
- Merged the location categories "South East England"  with "Scotland" for consistency

lizjegede committed:
Added content

Added two stories, resources, google form and added content to the about
us section
